torch/csrc/jit/passes/onnx/fixup_onnx_conditionals.cpp

```diff
@@ -0,0 +1,42 @@
+#include <torch/csrc/jit/passes/onnx/fixup_onnx_conditionals.h>
+
+namespace torch {
+namespace jit {
+
+namespace onnx{
+using namespace ::c10::onnx;
+}
+
+bool FixupONNXIfs(Block* block) {
+  bool has_nodes = false;
```

Comment:
It's a very narrow case, but could you double-check if this also fix cases with imprecated if blocks as well?